azure-pipelines.yml

```diff
@@ -8,13 +8,13 @@ jobs:
 - job: 'StyleCheck'
 
   pool:
-    vmImage: 'ubuntu-20.04'
+    vmImage: 'ubuntu-latest'
```

Comment:
I thought we are using GitHub actions? Is this file needed?

Reply:
@oruebel In general yes, but this repo does not have them setup. I will make a ticket for that, but this will get it updated till then. Talking with Ryan in our group chat, this was before the use of actions.